client/my-sites/comments/comment/comment-author.jsx

```diff
@@ -39,17 +39,6 @@ export class CommentAuthor extends Component {
 
 	storeLinkIndicatorRef = icon => ( this.hasLinkIndicator = icon );
 
-	commentHasLink = () => {
-		if ( typeof DOMParser !== 'undefined' && DOMParser.prototype.parseFromString ) {
-			const parser = new DOMParser();
-			const commentDom = parser.parseFromString( this.props.commentContent, 'text/html' );
-
-			return !! commentDom.getElementsByTagName( 'a' ).length;
-		}
-
-		return false;
-	};
-
```

Comment:
As a first step we can move this to the data-layer or in the reducer, whenever we receive a comment. If `contains_links` is not available, we can calculate this once instead of on each render.

Reply:
We'd be able to land ^^ decoupled from any endpoint work.

Reply:
@gwwar thanks, updated as suggested in b2c156ef0424a67c1c6b63a8b582e4a0d97e9614.